### PR TITLE
fix: formatting in shadow-tlb.h

### PR DESCRIPTION
**Pull request by diegonehab**
We are still adjusting our CI. Somehow, this formatting error went through.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Build/Test/Verify/Release
-on: [push, pull_request_target]
+on: [push, pull_request]
```
Comment (mpolitzer) [inline .github/workflows/build.yml:2]: Is this change security related?

Comment (vfusco) [reply .github/workflows/build.yml:2]: No, it was a misconfiguration. I misunderstood what the "context" meant. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

**Review by vfusco — APPROVED** [review]

**Review by mpolitzer — APPROVED** [review]

**Review by edubart — APPROVED** [review]
